CMakeLists.txt

```diff
@@ -786,6 +786,7 @@ add_subdirectory(${LIBRARIES}/filabridge)
 add_subdirectory(${LIBRARIES}/filaflat)
 add_subdirectory(${LIBRARIES}/filagui)
 add_subdirectory(${LIBRARIES}/filameshio)
+add_subdirectory(${LIBRARIES}/generatePrefilterMipmap)
```

Comment:
nit: would `envprefilter` be a reasonable alternative name?  So that we'd get a lib name that's parallel to `iblprefilter` (two seem to be serving same purpose for different maps).

Reply:
They're actually mostly doing the same thing, but one is using the cpu and one is using the gpu.